android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -335,9 +351,39 @@ public void initializeState() {
                     this, dataTypesArray, TimePeriod.ALL_TIME);
         }
 
+        setLoadedFeed(false);
+        setNewsItemsFeedCards(null);
         BraveSearchEngineUtils.initializeBraveSearchEngineStates(getTabModelSelector());
     }
 
+    public boolean loadedFeed;
+    public CopyOnWriteArrayList<FeedItemsCard> newsItemsFeedCards;
+    public int newsFeedScrollPosition;
+
+    public boolean isLoadedFeed() {
+        return loadedFeed;
+    }
+
+    public void setLoadedFeed(boolean loadedFeed) {
+        this.loadedFeed = loadedFeed;
+    }
+
+    public CopyOnWriteArrayList<FeedItemsCard> getNewsItemsFeedCards() {
+        return newsItemsFeedCards;
+    }
+
+    public void setNewsItemsFeedCards(CopyOnWriteArrayList<FeedItemsCard> newsItemsFeedCards) {
+        this.newsItemsFeedCards = newsItemsFeedCards;
+    }
+
+    public int getNewsFeedScrollPosition() {
+        return newsFeedScrollPosition;
```

Comment:
I can see that private  `newsFeedScrollPosition` is used only in `getNewsFeedScrollPosition/setNewsFeedScrollPosition`; there is a code which invokes `setNewsFeedScrollPosition` and there is no code which invokes `getNewsFeedScrollPosition()`.

Is `newsFeedScrollPosition` required ?

Reply:
removed they are not used

Reply:
thanks